deploy/common/deploy.sh

```diff
@@ -155,7 +155,7 @@ function get_cluster_zip {
     fi
 
     echo "Creating a new cluster"
-    export CLUSTER_JSON="{\"name\": \"$CLUSTER_NAME\", \"type\": \"$CLUSTER_TYPE\", \"main_image\": \"$CLUSTER_IMAGE\", \"central_api_endpoint\": \"$CLUSTER_API_ENDPOINT\", \"collection_method\": \"$COLLECTION_METHOD_ENUM\", \"admission_controller\": $ADMISSION_CONTROLLER $EXTRA_JSON}"
+    export CLUSTER_JSON="{\"name\": \"$CLUSTER_NAME\", \"type\": \"$CLUSTER_TYPE\", \"main_image\": \"$CLUSTER_IMAGE\", \"central_api_endpoint\": \"$CLUSTER_API_ENDPOINT\", \"collection_method\": \"$COLLECTION_METHOD_ENUM\" $EXTRA_JSON}"
```

Comment:
Note: if `ADMISSION_CONTROLLER` is true, `EXTRA_JSON` will be updated in `deploy/common/k8sbased.sh` with `, "admissionController": true`

Reply:
That's why it was doubled and removal fixed that.